src/hierarchy.rs

```diff
@@ -181,6 +216,48 @@ pub enum VarType {
     StdULogicVector,
 }
 
+impl fmt::Display for VarType {
```

Comment:
Again, `Dispplay` is for user facing output and `wellen` is a library. The user facing output should be left up to the app that is using `wellen`.